php/class-ad-layers-post-type.php

```diff
@@ -107,12 +107,14 @@ public function manage_edit_columns( $columns ) {
 	  		$columns['ad_layer_page_types'] = __( 'Page Type', 'ad-layers' );
 	  		$columns['ad_layer_post_types'] = __( 'Post Types', 'ad-layers' );
 	  		$columns['ad_layer_taxonomies'] = __( 'Taxonomies', 'ad-layers' );
+	  		$columns['ad_layer_terms'] = __( 'Terms', 'ad-layers' );
 	  		$columns['ad_layer_ad_units'] = __( 'Ad Units', 'ad-layers' );
 	  		$columns['ad_layer_priority'] = __( 'Priority', 'ad-layers' );
 
-	  		// Move date back to the end
+	  		// Remove Date
 	  		unset( $columns['date'] );
-	  		$columns['date'] = __( 'Date', 'ad-layers' );
+	  		unset( $columns['categories'] );
```

Comment:
Ad layers can be targeted by categories and tags. It was intentional to leave them in.